src/librbd/crypto/luks/EncryptionFormat.cc

```diff
@@ -10,6 +10,9 @@ namespace librbd {
 namespace crypto {
 namespace luks {
 
+template <typename I>
+const std::string EncryptionFormat<I>::CLONE_PASSPHRASE_PREFIX = "rbd_clone.";
```

Comment:
Nit: it would be good to have a review of this approach by an encryption SME. I personally think it wouldn't introduce any extra weakness, it would be good to have the confirmation.